db/seeds.rb

```diff
@@ -7,11 +7,11 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
 foodtrucks = [
-    ["Chicken & Rice Guys", "Food truck that specializes in
+    ["chicken & rice guys", "Food truck that specializes in
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.